demo-app/src/androidMain/kotlin/org/maplibre/compose/demoapp/demos/GmsLocationDemo.kt

```diff
@@ -32,19 +34,31 @@ object GmsLocationDemo : Demo {
     // this if _is_ a permission check Lint just doesn't know that
     @SuppressLint("MissingPermission")
     if (state.locationPermissionState.hasPermission) {
-      val locationProvider = rememberFusedLocationProvider()
+      val locationProvider = rememberFusedLocationProvider(LocationRequest(orientation = true))
       val locationState = rememberUserLocationState(locationProvider)
 
       LocationPuck(
         idPrefix = "gms-location",
-        locationState = locationState,
+        location = locationState.location,
+        bearing =
+          locationState.location?.let { location ->
+            val courseAccuracy = location.course?.inaccuracy ?: 180.degrees
+            val orientationAccuracy = location.orientation?.inaccuracy ?: 180.degrees
```

Comment:
```suggestion
            val courseAccuracy = location.course?.inaccuracy ?: Double.POSITIVE_INFINITY.degrees
            val orientationAccuracy = location.orientation?.inaccuracy ?: Double.POSITIVE_INFINITY.degrees
```

While `180.degrees` produces the correct result, `Double.POSITIVE_INFINITY.degrees` makes the intention more clear for the demo